lib/Runtime/Language/WAsmjsUtils.cpp

```diff
@@ -289,6 +289,9 @@ template<> Types RegisterSpace::GetRegisterSpaceType<AsmJsSIMDValue>(){return WA
             }
         }
 
+        // The offset currently carries the total size of the funcInfo after handling the last type
+        funcInfo->SetTotalSizeinBytes(offset);
+
```

Comment:
nit: This is some bug fix right? nothing to do with SIMD

Reply:
We used to assume simd to be the last type in asm.js/wasm. With this we recalculated the size in bytes.
Without simd, it makes more sense to cache the size at the time we calculate everything and just give it back afterward, without having to know which type is last nor duplicating the math.
See [AsmJsFunctionInfo::GetTotalSizeinBytes()'s implementation change for details](https://github.com/Microsoft/ChakraCore/pull/4541/commits/7806587638d62610232e23d66e263e212fbf697b#diff-3c5dbfb193356747438af5fa1e350b69L921)